deepeval/metrics/answer_relevancy/answer_relevancy.py

```diff
@@ -30,6 +31,13 @@ class AnswerRelvancyVerdict(BaseModel):
 
 
 class AnswerRelevancyMetric(BaseMetric):
+
+    _statements: ContextVar[List[str]] = ContextVar('statements', default=[])
```

Comment:
Is it possible for context variables to default to None? Since right now we default to none: https://github.com/confident-ai/deepeval/blob/main/deepeval/metrics/base_metric.py#L7

Also add `error` as well.